services/harmony/app/server.ts

```diff
@@ -91,7 +93,19 @@ function buildBackendServer(port: number, hostBinding: string): Server {
   app.get('/', ((req, res) => res.send('OK')));
   app.use(errorHandler);
 
-  return app.listen(port, hostBinding, () => appLogger.info(`Application backend listening on ${hostBinding} on port ${port}`));
+  let listener;
+  if (useHttps === 'true') {
```

Comment:
For safety sake maybe change this to
```
if (useHttps.toLowerCase() === 'true') {
```

Reply:
Or maybe just convert it to lowercase when you first set up the config from the env var

Reply:
I prefer checking for the exact string match - this is a config parameter we directly control so we don't need to be flexible on what we support unlike something like a query parameter from a user.